packages/financial-templates-lib/test/clients/ExpiringMultiPartyClient.js

```diff
@@ -39,7 +62,9 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
       let identifierWhitelist;
       let store;
       let identifier;
-      let convert;
+      let convertCollateral;
```

Comment:
This is the primary change made to the tests for the `ExpiringMultiPartyClient`: introduction of `convertCollateral` `convertSynthetic` and `convertPrice` which are used accordingly within the tests to validate that the EMP client works with all 3 combinations of decimals.